Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:12-alpine
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+RUN npm prune --production
+USER node
+ENV NODE_ENV=production
+ENV PORT=8080
+EXPOSE 8080
+CMD ["node", "dist/main.js"]
```

Comment:
The total size of the image is 369MB. To decrease image size should use a multistage build.
But, anyway, 369 is less than 500.

<img width="537" alt="Screenshot 2021-09-27 at 17 42 58" src="https://user-images.githubusercontent.com/85166028/134931538-11baf021-1dc0-4bd2-9c59-ad0e6204fa80.png">

Reply:
Unfortunately, I can't use multistage build, because I use Linux 2 platform.

Reply:
Guess, the reason could be in the alias in FROM section:
`FROM node:12-alpine AS build`
You could try to use index instead:
```
FROM node:12-alpine
# do some stuff
FROM node:12-alpine
COPY --from=0 /tmp /tmp
```